common/script/script-api/src/main/java/org/thingsboard/script/api/tbel/TbUtils.java

```diff
@@ -273,13 +273,13 @@ public static void register(ParserConfiguration parserConfig) throws Exception {
                 Long.class, boolean.class, boolean.class)));
         parserConfig.addImport("longToHex", new MethodStub(TbUtils.class.getMethod("longToHex",
                 Long.class, boolean.class, boolean.class, int.class)));
-        parserConfig.addImport("intLongToString", new MethodStub(TbUtils.class.getMethod("intLongToString",
```

Comment:
We can't BREAK existing APIs. They must be left as-is

Reply:
It's okay, these methods were introduced only in 3.7.1